### DISCUSSION TITLE
Can WT bg console-based text-editors?

### DISCUSSION THREAD

**Discussion by gamagan** [microsoft/terminal#13448] [Q&A] [answered]
Can Windows Terminal suspend applications to the background, and then unsuspend them?
Specifically, I'm interested in doing this for the text editor Micro `https://micro-editor.github.io/` or TextAdept `https://orbitalquark.github.io/textadept/`. That is, open some text to edit, suspend the editor in the bg and drop back to the terminal, run some commands, and then get back to the editor.

Thanks

**Comment by DHowett** [comment, marked as answer]
Unfortunately, not today. There's no support in the Windows platform for suspending console applications and returning control to the shell -- there's no real notion of job control for an attached console session.
